test/Microsoft.ML.Sweeper.Tests/TestSweeper.cs

```diff
@@ -0,0 +1,756 @@
+                            j = (val == 100) ? 0 : (val == 1000) ? 1 : 2;
+                        }
+                        else
+                        {
+                            Assert.True(false, "Wrong parameter");
+                        }
+                    }
+                    Assert.False(gridPoint[i][j]);
+                    gridPoint[i][j] = true;
+                }
+                Assert.True(gridPoint.All(bArray => bArray.All(b => b)));
+            }
+        }
+
+        [Fact(Skip = "The test is not able to load ldrandpl")]
+        public void TestNelderMeadSweeper()
+        {
+            var random = new Random(42);
+            var args = new NelderMeadSweeper.Arguments();
+            using (var env = new TlcEnvironment(42))
+            {
+                args.SweptParameters = new IComponentFactory<INumericValueGenerator>[] {
+                    ComponentFactoryUtils.CreateFromFunction(
+                        environ => new FloatValueGenerator(new FloatParamArguments() { Name = "foo", Min = 1, Max = 5})),
+                    ComponentFactoryUtils.CreateFromFunction(
+                        environ => new LongValueGenerator(new LongParamArguments() { Name = "foo", Min = 1, Max = 100, LogBase = true }))
+                };
+                var sweeper = new NelderMeadSweeper(env, args);
+                var sweeps = sweeper.ProposeSweeps(5, new List<RunResult>());
+                Assert.Equal(3, sweeps.Length);
+
+                var results = new List<IRunResult>();
+                for (int i = 1; i < 10; i++)
+                {
+                    foreach (var parameterSet in sweeps)
+                    {
+                        foreach (var parameterValue in parameterSet)
+                        {
+                            if (parameterValue.Name == "foo")
+                            {
+                                var val = Float.Parse(parameterValue.ValueText, CultureInfo.InvariantCulture);
+                                Assert.True(val >= 1);
+                                Assert.True(val <= 5);
+                            }
+                            else if (parameterValue.Name == "bar")
+                            {
+                                var val = long.Parse(parameterValue.ValueText);
+                                Assert.True(val >= 1);
+                                Assert.True(val <= 1000);
+                            }
+                            else
+                            {
+                                Assert.True(false, "Wrong parameter");
+                            }
+                        }
+                        results.Add(new RunResult(parameterSet, random.NextDouble(), true));
+                    }
+
+                    sweeps = sweeper.ProposeSweeps(5, results);
+                }
+                Assert.True(sweeps.Length <= 5);
+            }
+        }
+
+#if !PORTED
```

Comment:
Can we leave this code out until it is ported and building?  I dislike having "dead" code in the repo.